### DISCUSSION TITLE
Better handling for multiple dates

### DISCUSSION THREAD

**Discussion by loewexy** [jonaswinkler/paperless-ng#1177] [Feature Requests]
I have lots of documents containing multiple dates and paperless does not always find the right one.

My simple suggestions would be to have another button at the input which does not open a calendar but opens a dropdown with all dates recognized in the document. So that I can choose the right one faster.

The luxury option would be a pipette symbol button which allows you to click on the right date in the PDF viewer, but I guess this is way harder to implement.

**Comment by maeries** [comment]
Another suggestion would be to have some kind of date blocklist. My problem is that many documents contain my date of birth. So for me it would be sufficient to tell paperless to ignore that one specific date

**Reply by sbrunner** [reply]
=> PAPERLESS_IGNORE_DATES :-)
https://paperless-ng.readthedocs.io/en/latest/configuration.html
